alf/algorithms/rl_algorithm.py

```diff
@@ -272,9 +261,7 @@ def summarize_train(self, experience, train_info, loss_info, params):
 
         if self._debug_summaries:
             summary_utils.summarize_action(experience.action,
-                                           self.train_action_spec)
-            if not self.is_on_policy():
-                self.summarize_reward("training_reward", experience.reward)
+                                           self._action_spec)
```

Comment:
You simply changed to self._action_spec from self.train_action_spec. Is the spec still correct for experience.action?

Reply:
Yeah there is no easy solution here because we can't change experience.action.